web/portal/src/main/webapp/WEB-INF/conf/organization/idm-configuration.xml

```diff
@@ -160,9 +160,17 @@
                 <key><string>/</string></key>
                 <value><string>root_type</string></value>
               </entry>
+              <!-- exo.group.platform.identity.type will define the identity type of groups under /platform group.
+                Before meeds 1.0 , the value was '.platform' , starting meeds 1.0 the value is GROUP -->
               <entry>
-                <key><string>${exo.ldap.groups.rootGroup:/platform/*}</string></key>
-                <value><string>GROUP</string></value>
+                <key><string>/platform/*</string></key>
+                <value><string>${exo.group.platform.identity.type:GROUP}</string></value>
```

Comment:
I understand that this commit had made simplification of LDAP installation https://github.com/Meeds-io/gatein-portal/pull/16, but it introduced a change to root group of LDAP groups. In other terms, it will add all LDAP mapped groups under /platform. By making this change, the LDAP mapped groups will be moved to `/ldap-groups/`. Will this induce a modification in the structure of groups of existing installations using LDAP mapping ?

I think, the changes made in this PR and the PR https://github.com/Meeds-io/gatein-portal/pull/16 needs a detailed **Upgrade** instructions of different use cases. (@hbenali @bourasom FYI )

Reply:
Currently, all groups under /platform/* are of identity type GROUP. Thus with this change I wanted to keep this changed beahvior for customers using eXo platform 6.0+ 
For old customers, they will be able to restore the default value (.platform) and their users will be imported under their /platform/* groups.
If a customer has a group mapping, he will be able to define the group under which he will import all his groups. I put the default value to /ldap-groups/ and didn't want to use an existing group